functions/source/rest_client.py

```diff
@@ -70,6 +70,10 @@ def handler(event, context):
                 responseData['NetworkId'] = post_result['network_id']
 
             if event['ResourceProperties']['action'] == 'CREATE_WORKSPACES':
+                if 'default_cluster' in event['ResourceProperties'] and event['ResourceProperties']['default_cluster'] == 'True':
```

Comment:
```suggestion
                if event['ResourceProperties'].get('default_cluster') == 'True':
```

Reply:
This will throw a `KeyError` when it tries to evaluate the second conditional. python dictionary`.get` returns `None` as a default value, and doesn't raise an exception.